src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Story } from "@storybook/react/types-6-0";
+import { Title, Description, ArgsTable, Stories } from "@storybook/addon-docs";
+import Toast, { ToastProps } from "./Toast";
+import Button from "../Button";
+
+export default {
+  title: "Components/Feedback/Toast",
+  component: Toast,
+  args: {
+    appearance: "info",
```

Comment:
`appearance` は各コンポーネントで渡してるからおそらく不要です。
逆に今変な位置に出てるので `placement: "top-center"` が欲しいです！！

```suggestion
    placement: "top-center",
```